### ISSUE TITLE
Implement basic product detail page

### ISSUE THREAD

**Issue by gsidebo** [mitodl/mitxonline#32]
**Depends on #16**
Design (desktop): https://projects.invisionapp.com/d/main/#/console/21535972/455477372/preview
Design (mobile): https://projects.invisionapp.com/d/main/#/console/21535972/455488639/preview

PR #11 added Wagtail and some initial barebones models for the home page. We want implement the product detail page in Wagtail and render it in a way that matches the designs (for our first version of this, course runs will be the only product, and a single page will exist for each course no matter how many runs it has, so practically this can just be referred to as the course detail page for now)

**Acceptance criteria:**
- [ ] Fields added to the `ProductPage` (implemented in #28) to store new data for course pages: `length` (text), `effort`, (text), `prerequisites` (text), ~~`long_description` (rich text)~~ about (text), `what_you_learn` (text)
  - [ ] Unlike xPRO, `price` should be a wagtail field where the CMS user can enter text or a link (as it appears in [InVision](https://invis.io/RN11DSJOGXFD#/455477372_Course_Detail)). This will give a MicroMasters program like DEDP a place to explain their personal pricing scheme. 
- [ ] `CoursePage`s can be created that match the designs in desktop and mobile views
- [ ] Courses featured on the home page link to the relevant product detail page 

**Out of scope:**
- Video (a simple placeholder div can be used instead of a video in the template)
- Faculty section (separate issue)
- The "Enroll now" button does not need to do anything (separate issue – https://github.com/mitodl/mitxonline/issues/33)
- ~~Price field. Our first courses in mitxonline will be free and we do not need to add any db models/fields to keep track of price~~
- Expand/collapse functionality for sections of text (indicated by "More..." in the designs)

**Implementation details:**
- Using the [design](https://projects.invisionapp.com/d/main/#/console/21535972/455477372/preview) as an example, the "About this course" and "What you'll learn" contents will be represented by the `about` and `what_you_learn` fields. The titles of those sections can be hard-coded into the template.

**Comment by HamzaIbnFarooq** [comment]
Need view permissions on invisionapp previews

**Comment by pdpinch** [comment]
The course team has requested that instead of omitting Price, we include a link under Price, with the link text and the link configurable in the CMS. (The text will probably be `Varies` but I'm not sure the link will go yet). 

To be prepared for future courses with actual prices, the CMS model should also allow values like "$150". 

I've updated the issue description to reflect this.
